velox/docs/monthly-updates/september-2023.rst

```diff
@@ -0,0 +1,93 @@
+*********************
+September 2023 Update
+*********************
+
+
+Documentation
+=============
+
+* Add :doc:`Chapter 2 of programming guide </programming-guide/chapter02>`
+* Add blog post about `learning from optimizing try_cast <https://velox-lib.io/blog/optimize-try_cast.>`_
+
+
+Core Library
+============
+
+* Add BaseVector::containsNullAt API :pr:`6515`
+* Add ContainerRowSerde::compareWithNulls API :pr:`6419`
+* Add default memory reclaimer for exchange client :pr:`6759`
+* Add Operator::initialize API to handle customized operation init :pr:`6422`
+* Add Pacific/Kanton, Europe/Kyiv, America/Ciudad_Juarez time zones :pr:`6670`
+* Add support for Array/Map/RowVector::copyRanges from UNKNOWN source :pr:`6607`
+* Add support for HUGEINT in function signatures :pr:`6799`
+* Add support for prioritizing productive sources in Exchange :pr:`6459`
+* Add support for UNKNOWN types to VectorHasher::hash :pr:`6435`
+* Add support for writing sorted buckted hive table :pr:`6142`
+* Extend Type with isOrderable() and isComparable() :pr:`6770`
+
+
+Presto Functions
+================
+
+* Add bitwise_xor_agg aggregate function :pr:`6705`
```

Comment:
- Let's group scalar, aggregate and window functions together. Let's also group casts together.
- Let's add links to function names using :func: